File: 3467-transform-array-by-parity/3467-transform-array-by-parity.cs
public class Solution {
    public int[] TransformArray(int[] nums) {
        List<int> result=new List<int>();
        for(int i=0;i<nums.Length;i++){
            if(nums[i]%2==0){
                result.Add(0);
            }else{
                result.Add(1);
            }
        }
        result.Sort();
        return  result.ToArray();
    }
}

File: 3467-transform-array-by-parity/3467-transform-array-by-parity.ts
function transformArray(nums: number[]): number[] {
    let result=[]
    for(let i=0;i<nums.length;i++){
        if(nums[i]%2==0){
            result.push(0);
        }else{
            result.push(1);
        }
    }
    return result.sort((a,b)=>a-b);
};